hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -496,6 +496,14 @@ private FlinkOptions() {
           + "If set true, the names of partition folders follow <partition_column_name>=<partition_value> format.\n"
           + "By default false (the names of partition folders are only partition values)");
 
+  @AdvancedConfig
+  public static final ConfigOption<Boolean> POPULATE_META_FIELDS = ConfigOptions
+      .key(HoodieTableConfig.POPULATE_META_FIELDS.key())
```

Comment:
This is unnecessary, all the Hudi core write/table options can be applied automatically on Flink.

Reply:
> This is unnecessary, all the Hudi core write/table options can be applied automatically on Flink.

Understood, thank you